sympy/simplify/sqrtdenest.py

```diff
@@ -236,6 +236,18 @@ def _sqrtdenest0(expr):
         else:
             n, d = [_sqrtdenest0(i) for i in (n, d)]
             return n/d
+
+    if isinstance(expr, Add):
+        cs = []
+        args = []
+        for arg in ordered(expr.args):
```

Comment:
I only wanted to understand its role since the arguments of `Add` are automatically ordered. If you believe that `ordered` would result in a better order, then I think that is ok. But I would like to have a comment added so that the purpose of `ordered` would be explained. Otherwise this looks good to me.